### DISCUSSION TITLE
Terraform/aws support

### DISCUSSION THREAD

**Discussion by sginovker** [telekom-security/tpotce#865] [Q&A] [answered]
Has anyone gotten TPot 20.06 working on AWS? (I'm aware it's not officially supported) I tried both installing manually on an ec2 instance and using Terraform, but both times I have trouble accessing anything from the web dashboard (Kibana, etc won't open). Most of the medium articles and videos on AWS for TPot are with older versions, I'm wondering if anyone knows what broke or how to fix it.

**Comment by t3chn0m4g3** [comment, marked as answer]
If I remember correctly AWS had some problems with disabling IPv6 in `/etc/sysctl.conf`. In the issues users reported to getting it to work by removing the corresponding lines.

**Reply by sginovker** [reply]
Thanks! After a few reboots, looks better. Another note for anyone else trying to use the Terraform configs, my elasticsearch container couldn't start due to not enough RAM with a t3.large EC2 instance. I just boosted mine to t3.xlarge and looks good now
